brianlong/www.validators.app
# 177618323 historical sol prices

#### What's this PR do?
- Create model, services and scripts to gather daily and historical sol prices.

#### How should this be manually tested?
- Run tests
- Run scripts from `script/sol_prices` (commands at the top of the files)
- Check after each scripts if `log/sol_prices.log` has been updated
- Check `sol_prices` table if records has been created

#### What are the relevant tickets?
- https://www.pivotaltracker.com/n/projects/2177859/stories/177618323

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
